### DISCUSSION TITLE
Is SSO support on roadmap?

### DISCUSSION THREAD

**Discussion by LeoQuote** [langgenius/dify#3261] [Help] [answered]
If dify can support SSO, it would reduce the work of account management a lot.

I checked the github issues and found it might not be available in open sourced version ? https://github.com/langgenius/dify/issues/3074#issuecomment-2031838826

If someone like me contributed for this feature, would it be accepted?

**Comment by takatost** [comment, marked as answer]
Yep, SSO supports is available in our enterprise commercial version. You can contact business@dify.ai for more information.

**Reply by LeoQuote** [reply]
Is self serviced password reset on the roadmap? Would it be accepted?
